@here/olp-sdk-dataservice-read/lib/SummaryRequest.ts

```diff
@@ -0,0 +1,54 @@
+import { HRN } from "./HRN";
+
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+/**
+ * A class that prepare information for calls to get Statistics from CoverageAPI
+ */
+export class SummaryRequest {
+    private _catalogHrn: string | undefined;
+    private _layerId: string | undefined;
+
+    get catalogHrn(): string | undefined {
```

Comment:
could you please use getCatalogHrn() to align with C++?

Reply:
done